mmv1/third_party/terraform/services/storage/resource_storage_anywhere_cache_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+package storage_test
+
+import (
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/plancheck"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"testing"
+)
+
+func TestAccStorageAnywhereCache_basic(t *testing.T) {
```

Comment:
nit: _update to indicate this is an update test.

```suggestion
func TestAccStorageAnywhereCache_update(t *testing.T) {
```

Reply:
Ack